tests/e2e-pw/tests/cart-block/cart-block.merchant.spec.ts

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import { BlockData } from '@woocommerce/e2e-types';
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { editBlockPage } from '../../utils/navigation/navigation';
```

Comment:
To make the codebase cleaner, we should import the [function from the packages](https://github.com/woocommerce/woocommerce-blocks/blob/fa3f9856629dcbf77d12fc9064b34b9820574776/docs/contributors/e2e-guidelines.md/#L51-L52).

Reply:
This is a local util, it's not part of that package. Do you think we should make an alias for local packages?